bin/convertToStdVCF.py

```diff
@@ -0,0 +1,540 @@
+from __future__ import print_function
+
+import gzip
+import json
+import os
+import sys
+import textwrap
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+# Import tool for converting DKFZ VCF files to vcf files conforming to the specifications of the
+# standard VCF version:
+vcf_version = 'VCFv4.2'
+
+StdVCF_header_cols = [
+    # These first 8 are required
+    'CHROM', 'POS', 'ID', 'REF', 'ALT', 'QUAL', 'FILTER', 'INFO',
+    # Format is required when we specify genotype information, followed by patient-ID
+    # headernames added in convert()
+    'FORMAT']
+
+
+def read_meta_information(file):
+
+    def dict_without_comments(d):
+        return dict(filter(lambda (k, v): not k.startswith("__"), d.items()))
+
+    with open(file, "r") as f:
+        raw_infos = json.load(f)
+        return dict(map(lambda (k, v): (k, dict_without_comments(v)),
+                        dict_without_comments(raw_infos).items()))
+
+
+def open_maybe_compressed_file(filename, mode="r"):
+    """
+    Checks if file is compressed or not and opens it accordingly.
+
+    Checks if the last three letters equal ".gz" and uses gzip.open() or open() depending on the
+    outcome.
+
+    :param filename: Path to the file that is going to be opened.
+    :param mode: Uses read-mode as default.
+    :return: Returns the opened fileobject.
+    """
+    if filename[-3:] == ".gz":
+        #print("detected " + filename + " as gzip file", file=sys.stderr)
+        return gzip.open(filename, mode)
+    else:
+        #print("detected " + filename + " as non-zip file", file=sys.stderr)
+        return open(filename, mode)
+
+
+def convert_str_to_dict(str_to_convert, pair_separator, key_val_separator):
+    """
+    Converts a flat string with key/value pairs into a dictionary.
+
+    :param str_to_convert: String to be converted to a dict
+    :param pair_separator: String or character that separates the key-value-pairs from each other
+    :param key_val_separator: String or character that separates keys from values
+    :return: dictionary that was created from the string
+    """
+    pairs = str_to_convert.split(pair_separator) 
+    valid_pairs = filter(lambda pair: len(pair.split(key_val_separator)) == 2, pairs)
+    return dict(pair.split(key_val_separator) for pair in valid_pairs)
+
+
+def convert_dict_to_str(dict_to_convert, pair_separator, key_val_separator):
+    """
+    Converts a flat dictionary with just a single level of keys and values into a string.
+    Pair- and key/value-separators are configurable.
+
+    :param dict_to_convert: Dictionary that is to be converted
+    :param key_val_separator: String or character that should separate keys from values in the
+           string
+    :param pair_separator: String or character that should separate the key-value-pairs from each
+           other
+    :return: string that was created from the dictionary
+    """
+
+    return pair_separator.join(
+        key + key_val_separator + val
```

Comment:
ANNOVAR adds ";" to certain annotations, eg: "exonic;splicing". Since, ";" is an INFO field separator, this will cause an issue in downstream parsing. So, replace ";" in `var` to ",".  

`key + key_val_separator + val.replace(";", ",")`